R/emissions_profile_any_compute_profile_ranking.R

```diff
@@ -25,12 +25,19 @@
 emissions_profile_any_compute_profile_ranking <- function(data) {
   check_emissions_profile_any_compute_profile_ranking(data)
 
-  if (hasName(data, "profile_ranking")) {
-    out <- check_crucial_names(data, "grouped_by")
-  } else {
```

Comment:
@kalashsinghal, 

I'm confused. What's the final decision? 

Do we pre-compute `profile_ranking` in tiltIndicatorBefore and pass it to tiltIndicator as a column in the `co2` datasets (i.e. products or inputs)? Correct? 

If so, removing this line will re-compute `profile_ranking` inside the `emissions_profile*()` functions unless we also remove this line:

https://github.com/2DegreesInvesting/tiltIndicator/blob/a220fe46ef6a48406f719a823096a58d09cbdfa0/R/emissions_profile_any_at_product_level.R#L13

Reply:
@kalashsinghal, 

But then how do we avoid the problem that initiated the whole change? That is, how do we ensure that `profile_ranking` is calculated using a dataset that has products that exist in the entire `ecoinvent` dataset, instead of using a dataset that has products that only exist in europages?

Reply:
@maurolepore There are new datasets namely `emissions_profile_products_ecoinvent` and `emissions_profile_upstream_products_ecoinvent` which has all the ecoinvent products, and they will be ranked in tiltIndictor using the `emissions_profile_any_compute_profile_ranking` function that we have shifted. 

The old datasets namely `pctr_products` and `ictr_products` have products that only exists in `Europages`, and they can also be ranked using the same `emissions_profile_any_compute_profile_ranking` function, however we are not using them for calculating the final results because Anne thinks they needs to be reviewed. 

In either case, we don't need to calculate `profile_ranking` in tiltIndicatorBefore because we have separate datasets for both the cases. Does it make sense?